benchflow-dsl/src/test/scala/cloud/benchflow/dsl/package.scala

```diff
@@ -11,5 +11,6 @@ package object dsl {
   val BenchFlowLoadTestExample: String = BenchFlowTestExamplesFolder + "load/benchflow-test.yml"
 
   val BenchFlowExplorationUsersExample: String = BenchFlowTestExamplesFolder + "exploration/one-at-a-time/users/benchflow-test.yml"
+  val BenchFlowExplorationMultipleExample: String = BenchFlowTestExamplesFolder + "exploration/one-at-a-time/multiple/benchflow-test.yml"
```

Comment:
I thought it was clear since it is under the exploration folder. It means `multiple exploration` as in multiple dimensions. Other folders contain only one dimension.